### ISSUE TITLE
Slider no longer showing up

### ISSUE THREAD

**Issue by wirecuttr** [Nerwyn/service-call-tile-feature#19]
**Checklist:**

- [ Y] I updated to the latest version available
- [ Y] I cleared the cache of my browser

**Release with the issue:**
3.1.7

**Last working release (if known):**

**Browser and Operating System:**
Android/Companion app
Windows/Edge
Windows/Chrome

**Description of problem:**

I had this card with a slider and some buttons working fine in an earlier release:
https://github.com/Nerwyn/service-call-tile-feature/issues/3#issuecomment-1803077901
Things were working back in Nov/2023 when this issue 3 was closed
I'm not sure when the slide disappear unfortunately. Any idea?
**Javascript errors shown in the web inspector (if applicable):**

**Your Full Remote Config**

Shortening the card to just the slide also doesn't fix things
```
features:
  - type: custom:service-call
    entries:
      - type: slider
        icon: mdi:ev-station
        service: emporia_vue.set_charger_current
        target:
          entity_id:
            - switch.ev_charger
        range:
          - 6
          - 48
        step: 1
        value_attribute: charging_rate
        thumb: default
        label: ATTRIBUTE[charging_rate] A
        data:
          current: VALUE
type: tile
entity: switch.ev_charger
icon_tap_action:
  action: toggle
icon: mdi:ev-station

```
yields:
![image](https://github.com/Nerwyn/service-call-tile-feature/assets/40327874/70587c87-47d7-4563-bb15-e889791a9d0b)

**Additional information:**

**Comment by Nerwyn** [comment]
Did you recently update? The latest release was almost a month ago so it would be weird if the issue were to suddenly appear. I also see you're still using the deprecated string interpolation system for the label. It should now be `'{{ state_attr("switch.ev_charger", "charging_rate") }}'` with `unit_of_measurement: A`.

It looks like it's failing [here](https://github.com/Nerwyn/service-call-tile-feature/blob/main/src/classes/service-call-slider.ts#L125-L129) because it's expecting entity ID to be a string instead of an array. I should fix that.

**Comment by Nerwyn** [comment]
@wirecuttr can you try the latest alpha? It should fix this.

**Comment by wirecuttr** [comment]
It's not a new problem. I'm just getting around to logging the issue.

with 3.1.8-alpha.002 the slider shows up, but not correctly:
![image](https://github.com/Nerwyn/service-call-tile-feature/assets/40327874/120f97d4-43f9-46fe-887d-4d2f0247944b)

In terms of changing the label format I tried this: 
```
features:
  - type: custom:service-call
    entries:
      - type: slider
        icon: mdi:ev-station
        service: emporia_vue.set_charger_current
        target:
          entity_id:
            - switch.ev_charger
        range:
          - 6
          - 48
        step: 1
        value_attribute: charging_rate
        thumb: default
        label: {{ state_attr("switch.ev_charger", "charging_rate") }}
        unit_of_measurement: A
        data:
          current: VALUE
type: tile
entity: switch.ev_charger
icon_tap_action:
  action: toggle
icon: mdi:ev-station
```
yields this error:
![image](https://github.com/Nerwyn/service-call-tile-feature/assets/40327874/a14ab50e-cec5-44da-80d7-8b3cad7e8f94)

**Comment by Nerwyn** [comment]
You have to put single quotes around the template.

**Comment by wirecuttr** [comment]
> You have to put single quotes around the template.

Ah. Thanks for your help!
This is working:
```
features:
  - type: custom:service-call
    entries:
      - type: slider
        icon: mdi:ev-station
        service: emporia_vue.set_charger_current
        target:
          entity_id: switch.ev_charger
        range:
          - 6
          - 48
        step: 1
        value_attribute: charging_rate
        thumb: default
        label: '{{ state_attr("switch.ev_charger", "charging_rate") }}'
        unit_of_measurement: A
        data:
          current: VALUE
type: tile
entity: switch.ev_charger
icon_tap_action:
  action: toggle
icon: mdi:ev-station
```

Is it posible to alter the colour of the slide based on the attribue value?
I'm trying this, but it's not working:

```
        color:
          {% if state_attr("switch.ev_charger", "charging_rate") > 35 %}
            red
          {% else %}
            blue
          {% endif %}
```

**Comment by Nerwyn** [comment]
You have an extra set of brackets within the if statement, and should be using `--color` instead of `color`.

```yaml
--color: |
  {% if state_attr("switch.ev_charger", "charging_rate") > 30 %}
    red
  {% else %}
    blue
  {% endif %}
```

**Comment by Nerwyn** [comment]
Fixed in [3.1.8](https://github.com/Nerwyn/service-call-tile-feature/releases/tag/3.1.8)
